packages/backend/src/tokens/jwt.ts

```diff
@@ -148,31 +152,39 @@ export async function verifyJwt(
   }
 
   // Verify audience claim (aud)
-  if (typeof aud === 'string') {
-    if (aud !== audience) {
+  const audiences = audience && Array.isArray(audience) ? audience : [audience];
+  const shouldVerifyAudience = (!!audiences && audiences.length > 0) || !aud;
```

Comment:
If `audience` is `undefined`, then `audiences` will be `[undefined]`, so then `shouldVerifyAudience` would be `true` where I believe you intend for it to be `false` (if no `audience` is passed in for verification, then it stands to reason that it should not be verified, which would match the behavior of other verifications in this function). 

```suggestion
  const shouldVerifyAudience = (!!audience && audiences.length > 0) || !aud;
```

Reply:
You are correct. I have updated the code and add some tests.